index.html

```diff
@@ -0,0 +1,263 @@
+              web or data science.
+            </p>
+          </div>
+          <div class="border-box">
+            <i class="fas fa-user-graduate"></i>
+            <h3>Scholarship</h3>
+            <p>
+              Scholarship helps you to achieve your goals and many scholarship
+              here you can choose from them.
+            </p>
+          </div>
+        </div>
+        <div class="learn-more">
+          <a class="learn btn btn-primary" href="#" rel="noreferrer noopener"
+            >More Information</a
+          >
+        </div>
+        <div class="see-det">
+          <h3 class="see-de h3">See All Courses</h3>
+        </div>
+      </div>
+    </section>
+    <section class="feature-speak col-12">
+      <h3 class="feature h3">
+        Featured Speakers
+        <span class="under-ll"></span>
+      </h3>
+      <div class="speaker-info col"></div>
+      <button
+        class="butt"
+        type="submit"
+        id="showHideButton"
+        onclick="showImage();"
+      >
+        MORE <i class="fas fa-angle-down"></i>
+      </button>
+      <div id="more-detail">
+        <div class="detail showHide">
+          <img
+            class="speaker-img img-sp"
+            src="./src-image/salma.jpg"
+            alt="salima"
+          />
+          <h3 class="speaker-name spe-na">Salima Ikram</h3>
+          <p class="speaker-desc spea-de">
+            Distinguished University Professor of Egyptology at The American
+            University in Cairo
+          </p>
+          <p class="speaker-mor spea-more">
+            worked as an archaeologist in Turkey, Sudan, Greece and the United
+            States
+          </p>
+        </div>
+        <div class="detail showHide">
+          <img
+            class="speaker-img img-sp"
+            src="./src-image/nagw.jpg"
+            alt="nagw"
+          />
+          <h3 class="speaker-name spe-na">Nagwa Sherif</h3>
+          <p class="speaker-desc spea-de">
+            Architectural Engineering at The American University in Cairo
+          </p>
+          <p class="speaker-mor spea-more">
+            She holds a PhD degree from University of Montreal, Canada in 1984
+          </p>
+        </div>
+      </div>
+    </section>
+    <section class="partner border">
+      <h3 class="partner1 h3">
+        Partner
+        <span class="under-lll"></span>
+      </h3>
+      <h4 class="omzilla col">mozilla</h4>
+      <h4 class="googl col">Google</h4>
+      <h4 class="never col">NAVER</h4>
+      <footer class="foot border-0">
+        <div class="logo-de1">
+          <img
+            class="logo-image1 img-thumbnail"
+            src="./src-image/logo-image.png"
+            alt="logo design"
+          />
+          <h2 class="title-l1">
+            Creative<br />
+            commons Egypt
+          </h2>
+          <p class="date-t1">
+            2021 Creative Common Egypt<br />Some Rights Reserved
+          </p>
+        </div>
+      </footer>
```

Comment:
**Suggested Changes**
- [ ] The footer should be centered as in the design. Yours is currently left-aligned. Let us make these chnages